app/components/dashboard/run/left-panel/component.js

```diff
@@ -361,38 +340,44 @@ export default Component.extend(FullScreenMixin, {
           window.location.assign(url + '?token=' + token + '&taleFormat=' + format);
         },
         
-        openPublishModal(tale) {
-            const jwt = this.getDataONEJWT();
-            if (!jwt) {
-                // reroute to auth
-                $('#dataone-auth-modal').modal('show');
-                return;
-            }
-            
-            this.set('dataoneJWT', jwt);
+        authenticateD1(taleId, coordinatingNode) {
```

Comment:
I may be misunderstanding something, but `authenticateD1` seems to be called before I select the member node to publish to.  This means that the coordinatingNode is always the staging node, since the dropdown defaults to the first entry (dev). As discussed on the dev call, this would work since the JWT is raw ORCID and identical for stage/prod. But, for this to work as intended it seems that the user would need to select the member node before authenticating, right?

Reply:
`authenticateD1` gets called when the user [clicks Continue to DataOne Login](https://github.com/whole-tale/dashboard/blob/ca4c565459d38c532ac19e2557d4569ce1c31a20/app/components/dashboard/run/left-panel/template.hbs#L102) after checking the token endpoint. I took out the call to the publish endpoint and confirmed that the correct nodes are checked on development and on production.

In the link above, `selectedRepository` is set each time the repository is changed, and should be current when this code is hit.

Reply:
Clearly user error on my part -- maybe a stale dashboard build.  I see now that the login prompt appears after I hit "Publish" (after selecting the target node) and that the correct CN is checked for the token.  During my testing, the login prompt was appearing when I clicked the "Publish Tale..." menu, but that's changed.